api/external/applications/applications.proto

```diff
@@ -181,8 +196,12 @@ message UpdateDisconnectedServicesConfigReq {
   string threshold = 2;
 }
 
+message GetDisconnectedServicesConfigReq{}
 message UpdateDisconnectedServicesConfigRes {}
 
+message GetDeleteDisconnectedServicesConfigReq{}
```

Comment:
I know I'm missing context but this message name is ambiguous-- am I getting deleted services? am I deleting disconnected services?

Reply:
OK I figured it out after a few more files-- fwiw, I think the addition of the word `Job` might help, but I know it's already an ungainly thing